handsontable/test/__mocks__/intersectionObserverMock.js

```diff
@@ -0,0 +1,9 @@
+export class IntersectionObserverMock {
+  observe() {
+    return null;
+  }
+
+  unobserve() {
+    return null;
+  }
```

Comment:
```suggestion
  observe() {}
  unobserve() {}
```
The MDN says that the methods return `undefined`, so I guess it'd be good to follow that rules for mock.